From 3537ef196667be116360ff69507be66f486c56b8 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Mon, 17 Oct 2022 08:40:14 +0800
Subject: [PATCH] Revert "Linux 5.10.148"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit 3783e64fee4a624f3ed1d7d6ae630890922edb7b.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c40acf09ce29..24110f834775 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 148
+SUBLEVEL = 147
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

